lib/screens/home_screen.dart

```diff
@@ -1,39 +1,195 @@
+// Copyright 2021 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
 import 'package:flutter/material.dart';
+import 'package:galacticcoders_website/screens/about_us_screen.dart';
+import 'package:galacticcoders_website/screens/blog_screen.dart';
+import 'package:galacticcoders_website/screens/component_screen.dart';
+import 'package:galacticcoders_website/screens/contact_us_screen.dart';
+import 'package:galacticcoders_website/screens/projects_screen.dart';
+import 'package:galacticcoders_website/screens/events_screen.dart';
+
+void main() {
+  runApp(const HomeScreen());
+}
 
-class HomeScreen extends StatelessWidget {
+class HomeScreen extends StatefulWidget {
```

Comment:
Just a quick note:
Stateful widgets are awesome! Most widgets I try to keep stateless and let a state management system handle how things are rebuilt and presented.

However we have to separate things for a second, app state using BloC for instance (https://pub.dev/packages/flutter_bloc) is HIGHLY recommended for that. If a widget should be able to do something simple such as changing color, toggling something simple or produce an animation you SHOULD absolutely use a stateful widget.

Nothing wrong with this solution, this is a discussion-comment